ci_framework/roles/polarion/vars/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# While options found within the vars/ path can be overridden using extra
+# vars, items within this path are considered part of the role and not
+# intended to be modified.
+
+# All variables within this role should have a prefix of "cifmw_polarion"
+
+cifmw_polarion_jump_repo_dir: "{{ ansible_user_dir }}/jump"
+cifmw_polarion_jump_result_dir: "{{ ansible_user_dir ~ '/ci-framework-data' }}/artifacts/tempest/"
```

Comment:
Wdyt about reading from `hostvars` instead of setting a default value.

By default this could be empty to support passing of a customized value.

Reply:
yes, usage of basedir seems better to me